logging/logging.go

```diff
@@ -0,0 +1,43 @@
+// Package logging provides a logger, as well as configuration.
+package logging
```

Comment:
I put it in a separate package so we were fine when we move to client-server.

And I prefer more, smaller packages.